models/Asset/Image/Thumbnail.php

```diff
@@ -321,7 +321,11 @@ public function getImageTag(array $options = [], array $removeAttributes = []):
         }
 
         if (empty($altText) && (!isset($options['disableAutoAlt']) || !$options['disableAutoAlt'])) {
-            if ($image->getMetadata('alt')) {
+
+            $altPrefix = Pimcore\Config::getSystemConfiguration('assets')['metadata']['alt_prefix'];
+            if (!empty($altPrefix) && $image->getMetadata($altPrefix.'.alt')) {
+                $altText = $image->getMetadata($altPrefix.'.alt');
```

Comment:
@kingjia90 I would not define it as prefix but just make the whole metadata attribute name configurable. If someone wants to use a totally different field name it would be ok, so this makes it more flexible.

In addition to that we should do the same approach for `title` and `copyright`.